apps/ui/lev-help.md

```diff
@@ -0,0 +1,72 @@
+know.
+
+The LEV service is available 24/7, with support provided during business hours
+(8am to 6pm, Monday to Friday).
+
+## Reporting a problem with this website
+### Initial report
+If there is a technical issue preventing you from using the website, HMPO users
+must raise an incident via the ITNow Service Desk, or via the HMPO Service Desk
+for those HMPO End Users who are using residual AdminIT.
+
+Non-HMPO users can raise a ticket with their own Service Desk support,
+who will contact the Home Office IT helpdesk.
+
+Non-government users should contact contact the Home Office IT helpdesk directly
+on 0845 000 0050 or by email at [ITnowServiceDesk@homeoffice.gov.uk](ITnowServiceDesk@homeoffice.gov.uk)
+to report problems.
+
+Triage will be performed by our level-2 support. All calls should be assigned
+to this team by the Service Desk and are dealt with in priority order.
+
+### If that doesn't work
+If you have not received a response within 5 working days, contact [lev-service@digital.homeoffice.gov.uk](mailto:lev-service@digital.homeoffice.gov.uk)
+with the ITNow Service Desk incident number.
+
+## Reporting a problem with your account
+If there is a problem with your user account, for example, you have the
+incorrect level of permissions or are unable to access data you need to see,
+please report the issue to:
+- HMPO Service Desk - for HMPO users
+- Service Desk Support - for non-HMPO users
+- Home Office IT helpdesk - for non-government users
+
+## Reporting inaccurate data on this website
+For data referrals and inaccurate records, please contact Jo Edkins on 0151 471
+4545 with a description of the inaccuracy and the page or record you found it on.
+
+## Reporting accessibility problems with this website
+We're always looking to improve the accessibility of this website.
+If you find any problems not listed on this page or think we're not meeting
+accessibility requirements, contact:
+- HMPO Service Desk - for HMPO users
+- Service Desk Support - for non-HMPO users
+- Home Office IT helpdesk - for non-government users
+
+For more information on accessibility, including what we are doing to meet our
+obligations, please visit our [Accessibility Statement](accessibility-statement).
+
+## Service Level Agreement (SLA)
+As mentioned above, problems are dealt with in priority order, depending on
+severity.
+
+<ol class="govuk-width-container">
+<li>A problem causing complete loss of service or loss of performance which
+renders the application unusable for the majority of users, and for which there
+is no immediate workaround, is considered a priority 1 incident. These incidents
+will be responded to within 4 hours.</li>
+<li>Loss of service for a moderate number of users, and for which there is no
+immediate workaround, is a priority 2 incident. These incidents will be
+responded to within 8 hours.</li>
+<li>Loss of service for a small number of users, or for a moderate number of
```

Comment:
Full-stop not required before parenthesis...also capitalised W probably too. Also applies to list item below.

Reply:
Amended now.